app/components/edit-reminder.js

```diff
@@ -24,6 +24,11 @@ export default Ember.Component.extend({
         record.save();
       })
     },
+    reset(){
+      this.element.children[0].textContent = this.model.data.title;
```

Comment:
I tried rollbackattributes and it didn't work and I think it had something to do with that I don't actually ever touch or change anything about the model until the user clicks the save button